.circleci/config.yml

```
@@ -8,6 +13,59 @@ jobs:
+              </servers>
+            </settings>
+            EOL
+      - run:
+          name: Release All
+          command: |
+            cat ~/.m2/settings.xml
+            mvn deploy -DskipTests -Prelease
+          environment:
+            GPG_TTY: /dev/tty
+            MAVEN_GPG_PASSPHRASE: 123456
           command: mvn test sonar:sonar
```

YAMLlint (1.37.1)

[error] duplication of key "command" in mapping

(key-duplicates)